### DISCUSSION TITLE
11.0.5

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#13290] [General]
https://www.nuget.org/packages/Avalonia/11.0.5

**Full Changelog**: https://github.com/AvaloniaUI/Avalonia/compare/11.0.4...11.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/11.0.5'>11.0.5</a>.</em>

**Comment by Sigrec** [comment]
Is there a way to center the content in a ItemsRepeater when using WrapLayout, setting the Alignment of the ItemsRepeater doesn't work? Something similar to UniformGridLayout's ItemsJustification="Center", I would use UniformGridLayout but whenever I search (clear the list then apply the flittered list) it sometimes doesn't render TextBlock height correctly when the Height is not explicitly set and it needs to calculate the height based on the wrapped text, but this issue isn't present with WrapLayout (I assume there is just a issue trying to dynamically load items in a UniformGrid).

**Resolved**
Looks like that happens because I was using a DockPanel switching to StackPanel works fine now

**Reply by timunie** [reply]
@Sigrec do you really think that this is the right place to ask such a question? I mean you're allowed to do, but there is a high chance it's overseen. 

Glad you found a solution 🙂

**Reply by Sigrec** [reply]
I put it here cause I didn't think it was a big enough issue to create a bug ticket for.

**Reply by timunie** [reply]
Not a bug ticket 😉. A Q&A (find it in Discussion) would be the right place and of course our telegram chat.

**Comment by llfab** [comment]
It seems @Gillibald fixed https://github.com/AvaloniaUI/Avalonia/issues/11804 with 11.0.5. At least in my app the text measurement is consistent now independent of which one is visible on startup. Since the issue is still marked as open you may want to double check though from your perspective.
